### PR TITLE
More work towards Cloud 6 support

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -823,53 +823,53 @@ function onadmin_set_source_variables()
...
+            CLOUDSLE12DISTISO="SUSE-SLE12-CLOUD-5-COMPUTE-x86_64*Media1.iso"
             CLOUDLOCALREPOS="SUSE-Cloud-5-devel"
         ;;
         develcloud6)
             CLOUDDISTPATH=/ibs/Devel:/Cloud:/6/images/iso
             [ -n "$TESTHEAD" ] && CLOUDDISTPATH=/ibs/Devel:/Cloud:/6:/Staging/images/iso
             CLOUDCOMPUTEPATH=$CLOUDDISTPATH
-            CLOUDDISTISO="SUSE-OPENSTACK-CLOUD-SLE11-6-$arch*Media1.iso"
-            CLOUDCOMPUTEISO="SUSE-SLE12-CLOUD-6-COMPUTE-x86_64*Media1.iso" # FIXME: not existing
+            CLOUDSLE11DISTISO="SUSE-OPENSTACK-CLOUD-SLE11-6-$arch*Media1.iso"
```
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: SLE11? Is this temporary? or is there still some SLE11 involved in Cloud 6?

Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: @toabctl it's temporary; the current plan is to slowly phase out the SLE11 bits. But until it's done, we need a SLE11 ISO.

**Comment by vuntz** [comment]
This depends on https://github.com/crowbar/crowbar/pull/2094.

**Comment by vuntz** [comment]
Forgot to mention: I still need to patch at least barclamp-provisioner to reflect the updated list of repos.

**Comment by aspiers** [comment]
:+1: if shell functions are also renamed accordingly.

**Comment by vuntz** [comment]
https://github.com/crowbar/barclamp-provisioner/pull/413 unbreaks provisioning of nodes.

**Comment by vuntz** [comment]
@aspiers did that in a new commit. Or do you prefer me to fix an older commit?

**Comment by aspiers** [comment]
@vuntz That's good enough for me, thanks :)

**Comment by bmwiedemann** [comment]
:+1:

**Comment by jdsn** [comment]
:+1:

**Pull request merged**